demo/site/src/common/components/Breadcrumbs.tsx

```diff
@@ -1,24 +1,48 @@
 "use client";
+import { createSitePath } from "@src/util/createSitePath";
 import { Fragment } from "react";
 
 import { GQLBreadcrumbsFragment } from "./Breadcrumbs.fragment.generated";
 import * as sc from "./Breadcrumbs.sc";
 import { GridRoot } from "./Breadcrumbs.sc";
 
-const Breadcrumbs = ({ name, path, parentNodes }: GQLBreadcrumbsFragment) => {
+type BreadcrumbsProps = GQLBreadcrumbsFragment & {
+    scope: {
+        language: string;
+    };
```

Comment:
This should be added to the breadcrumbs fragment

Reply:
so true, done